cli/bpmetadata/path.go

```diff
@@ -0,0 +1,85 @@
+package bpmetadata
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+const (
+	regexEx  = ".*/(examples/.*)"
+	regexMod = ".*/(modules/.*)"
+)
+
+func isPathValid(path string) (bool, error) {
+	_, err := os.ReadFile(path)
+	if err != nil {
+		return false, fmt.Errorf("unable to read file at the provided path: %v", err)
+	}
+
+	return true, nil
+}
+
+func getExamples(configPath string) ([]BlueprintMiscContent, error) {
+	re := regexp.MustCompile(regexEx)
```

Comment:
With must compile we can move this to a global
https://pkg.go.dev/regexp#MustCompile

Reply:
> The expression string is a global variable and we can't do MustComplie outside of a func body

What was the issue with using MustCompile outside func as a global var? The `Must*` are meant to be used for global init as mentioned in the [godoc](https://pkg.go.dev/regexp#MustCompile).